cmd/cloudshell_open/appfile.go

```diff
@@ -118,19 +141,30 @@ func promptEnv(list map[string]env) ([]string, error) {
 	// field and prompt the questions as they appear in the app.json file as
 	// opposed to random order we do here.
 	for k, e := range list {
-		var resp string
-		if err := survey.AskOne(&survey.Input{
-			Message: fmt.Sprintf("Value of %s environment variable (%s)",
-				color.CyanString(k),
-				color.HiBlackString(e.Description)),
-			Default: e.Value,
-		}, &resp,
-			survey.WithValidator(survey.Required),
-			surveyIconOpts,
-		); err != nil {
-			return nil, fmt.Errorf("failed to get a response for environment variable %s", k)
+
+		if e.Generator == "secret" {
```

Comment:
where's the validation that validates that `generator` and `value` cannot be set at the same time?

Reply:
Added in the parsing step